common/display/vblankeventhandler.cpp

```diff
@@ -87,14 +97,19 @@ void VblankEventHandler::HandlePageFlipEvent(unsigned int sec,
   int64_t timestamp = ((int64_t)sec * kOneSecondNs) + ((int64_t)usec * 1000);
   IPAGEFLIPEVENTTRACE("HandleVblankCallBack Frame Time %f",
                       static_cast<float>(timestamp - last_timestamp_) / (1000));
+  int64_t vperiod = timestamp - last_timestamp_;
   last_timestamp_ = timestamp;
 
   IPAGEFLIPEVENTTRACE("Callback called from HandlePageFlipEvent. %lu",
                       timestamp);
   spin_lock_.lock();
   if (enabled_ && callback_) {
-    callback_->Callback(display_, timestamp);
+     if(abs(vperiod - vperiod_) > (13333333 - 11111111))
```

Comment:
replace with Macro like 90Hz_PERIOD_NANOS and 75Hz_PERIOD_NANOS, the number is hard to understand